pipelines/data-generation/geni/machines

```diff
@@ -0,0 +1,5 @@
+node0.clapp-fran1307.schedock-PG0.clemson.cloudlab.us ansible_user=fran1307 ansible_become=true
```

Comment:
since this file gets generated as part of the execution of the pipeline, we need to add it to `.gitignore`